webapp/components/form/Dropdown/Dropdown.js

```diff
@@ -123,6 +125,7 @@ Dropdown.propTypes = {
   selection: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.string]),
   sourceElement: PropTypes.object, // Used to calculate the size of the autocomplete-dialog if available, otherwise the dropdownRef.current is used
   validation: PropTypes.object,
+  customItemsFilter: PropTypes.func,
```

Comment:
what is this? if you need to pass different items, filter them outside and pass them to items prop

Reply:
I realized that's used by InputChips. 
InputChips is responsible to reject the items. Add the logic there

Reply:
Thanks! I felt the pain when I put this prop into this component, take a look on the next commit to check if this approach is better,

Reply:
that's much more consistent now. thanks!